pysat/tests/test_instrument.py

```diff
@@ -319,8 +333,8 @@ def test_basic_instrument_load_no_input(self):
                              [('fname', 'have multi_file_day and load by file'),
                               (None, 'is not supported with multi_file_day')])
     def test_basic_instrument_load_by_file_and_multifile(self, load_in, verr):
```

Comment:
I think this test expanded since originally named. Perhaps, 
```suggestion
    def test_instrument_load_errors_with_multifile(self, load_in, verr):
```